internal/cmn/schema/dag.schema.json

```
@@ -1132,6 +1136,18 @@
     "step": {
       "type": "object",
       "additionalProperties": false,
+      "allOf": [
```

Biome (2.4.10)

[error] The key allOf was already declared.

(lint/suspicious/noDuplicateObjectKeys)